install_files/ossec-server/var/ossec/etc/ossec.conf

```diff
@@ -83,14 +83,9 @@
     <command>df -h</command>
   </localfile>
 
- <localfile>
-    <log_format>syslog</log_format>
-    <location>/var/log/kern.log</location>
-  </localfile>
-
   <localfile>
     <log_format>syslog</log_format>
-    <location>/var/log/cron-apt/log</location>
+    <location>/var/log/kern.log</location>
```

Comment:
That's a f'in weird merge, but I think I understand what happened. Seems ok to me.